src/platform/macos/events_loop.rs

```diff
@@ -295,7 +295,6 @@ impl EventsLoop {
         // FIXME: Document this. Why do we do this? Seems like it passes on events to window/app.
         // If we don't do this, window does not become main for some reason.
         match event_type {
```

Comment:
This `match` block can be replaced with `appkit::NSApp().sendEvent_(ns_event);` now.